published/012-2018-Oct-2.md

```diff
@@ -0,0 +1,63 @@
+## Welcome to the IPFS Weekly. 👋
+
+The [InterPlanetary File System (IPFS)](https://ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identity. IPFS enables the creation of completely distributed applications. It aims to make the web faster, safer, and more open. Since that’s a pretty large scope, we track development across the ecosystem in this weekly dispatch.
+
+Looking to get involved? Click on some of the links below, see what we’re up to on [GitHub](https://github.com/ipfs), or join us on [IRC](https://webchat.freenode.net/?channels=ipfs_) (#ipfs on the Freenode network).
+
+Want this update in your inbox? [Subscribe to our weekly newsletter!](https://tinyletter.com/ipfsnewsletter)
+
+Here are some of the highlights since the last IPFS Weekly.
+
+## The latest
+
++ Meet folks from Protocol Labs next Thursday at our Tech Drinkup in Glasgow. [Register for the event here](https://www.eventbrite.com/e/protocol-labs-tech-drinkup-tickets-50860367729).
++ See IPFS in action in the first episode of [Around the block](https://lab.paratii.video/play/GLdcTbPq9fXb), a docu-series of interviews featuring some of the most intriguing characters & stories “in crypto-land” by Paratii Video. The best part? Their content is streamed via a p2p app called [the Lab](https://lab.paratii.video). The Lab is a repository for high-quality crypto content that doesn’t spy on you and relies on IPFS. It’s also is open source and you can distribute the series yourself through embedding.
++ Introducing [IpfsCloud: A Decentralised Cloud Storage Platform](https://medium.com/ipfscloud/ipfscloud-a-decentralised-cloud-storage-platform-caf000173cd4) – A decentralised cloud storage platform, or the IPFS alternative to Google Drive 
++ Check out the recent [Filecoin demos with Chinese subtitles](https://www.youtube.com/watch?v=x145fNl6uKU). Many thanks to [Neo Ge](https://twitter.com/NeoGe_IPFSMain), of IPFS Main, for the translation.
+
+## IPFS in the wild
+*Do you follow [IPFS on Twitter](https://twitter.com/IPFSbot)? For the latest mentions of IPFS in the news, check our Twitter feed or see the [latest articles on Awesome IPFS](https://awesome.ipfs.io/categories/articles/).* 
```

Comment:
Another item for this section are the videos from Our Networks: https://ournetworks.ca/conference/recorded-talks/

Particularly this one where they explain how they live streamed the whole event over IPFS: https://ournetworks.ca/video/?ipfs=QmNtSy8JxmURQzFmW5r44JV6T3KfatLbEFFvePG1a1xK8i